core/common/src/main/java/alluxio/underfs/Fingerprint.java

```diff
@@ -118,6 +118,15 @@ public static Fingerprint parse(String input) {
     return new Fingerprint(tagMap);
   }
 
+  /**
+   * Checking if the fingerprint object was generated from an INVALID_UFS_FINGERPRINT.
```

Comment:
NIT: "Checks"

this is the convention for method comments.

Reply:
done